Makefile

```diff
@@ -414,6 +424,17 @@ release-build:
 	cd hack && $(GO) run release.go --templateDir ./templates/values/ --outputDir ../deployment/network-operator/
 	cd hack && $(GO) run release.go --templateDir ./templates/config/manager --outputDir ../config/manager/
 
+.PHONY: helm-update-nic-configuration
+helm-update-nic-configuration: $(HELM) $(HELM_CHART_DEP_CHECKDIR)
+	rm -rf ./build/helmcharts/nic-configuration-operator-chart
+	@NIC_CONF_VERSION=$$(yq '.nicConfigurationOperator.version' hack/release.yaml | sed 's/^v//'); \
```

Comment:
What about changes in `nicConfigurationConfigDaemon` ?

Reply:
I took here an assumption that the version of the operator match the chart version.
We don't have currently support for chart versions in release.yaml.